### DISCUSSION TITLE
EFK stack

### DISCUSSION THREAD

**Discussion by RicoXHub** [fluent/fluentd#4290] [Q&A]
I'm trying to find information about Fluentd if is compatible with  Kibana and Elasticsearch version 8.9.1. Currently using image below, will it work or will there be any issues?

containers:
      - name: fluentd
        image: fluent/fluentd-kubernetes-daemonset:v1.11.5-debian-elasticsearch7-1.1

**Comment by kenhys** [comment]
If you want to use with Elasticsearch version 8.x, it may better to use the image for 8.x series such as:
fluent/fluentd-kubernetes-daemonset:v1.16-debian-elasticsearch8-1
